packages/api-server/src/plugins/withFunctions.ts

```diff
@@ -105,7 +105,10 @@ const lambdaRequestHandler = async (
 const withFunctions = async (app: FastifyInstance, apiRootPath: string) => {
   // Add extra fastify plugins
   app.register(fastifyUrlData)
-  app.register(fastifyRawBody)
+
+  // In Fastify v4 must await the plugin registration to
```

Comment:
What is it that needs to await?
Should it be `// In Fastify v4 we must ...` or perhaps `// In Fastify v4 Fastify must ...`? Or something else entirely?

Reply:
Changed to 

```
  // Fastify v4 must await the fastifyRawBody plugin
  // registration to ensure the plugin is ready
  await app.register(fastifyRawBody)
```